### DISCUSSION TITLE
📢 Plans to archive the DICOM server project

### DISCUSSION THREAD

**Discussion by mmitrik** [microsoft/dicom-server#3401] [General]
After more than 3 years, our team is planning to archive the DICOM server project to allow us to focus on delivering customer value to our [managed service offering on Azure](https://learn.microsoft.com/en-us/azure/healthcare-apis/dicom/overview).  Our current plans are to archive this project on **April 30th, 2024**.

We are genuinely interested in your feedback on our plans – please leave us comments here if you have any.  We also want to understand the use cases you have that led you to choose to self-host the DICOM server vs. use the managed service offering.  Learning from the community can help us prioritize the right investments in the future.  

We realize that for some, this project served as a place to connect with our development team and to learn about upcoming capabilities.  Going forward, we’ll be directing the community to leave feedback on the DICOM service in our [community forum on the Azure feedback site](https://feedback.azure.com/d365community/forum/caab15ac-311a-ed11-b83e-000d3a4d97ea).  For new feature developments in the DICOM service, our [release notes](https://learn.microsoft.com/en-us/azure/healthcare-apis/release-notes) will be the best source of information.

**Comment by kyleburnett** [comment]
Our team was hoping to use this for local development, sort of like how the [Azurite emulator can be used for local Azure Storage development](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite). Do you know of any software that could be used for local DICOMweb development (with the intention to use Azure DICOM service in production)?

**Comment by Medical-Innovation-Technology-dev** [comment]
The template to deploy on Azure, will still available? I found usefull this project ='(
